daemon/graphdriver/vfs/driver.go

```diff
@@ -24,6 +26,7 @@ func Init(home string, options []string) (graphdriver.Driver, error) {
 	return graphdriver.NaiveDiffDriver(d), nil
 }
 
+// Driver holds information about the driver, home directory of the driver.
```

Comment:
Driver implements graphdriver.ProtoDriver.
It uses only basic vfs operations.
In order to support layering, files are copied from the parent layer into the new layer. There is no copy-on-write support.
Driver must be wrapped in `NaiveDiffDriver` to be used as a `graphdriver.Driver`